src/app/shared/mock-data.service.ts

```diff
@@ -186,6 +192,15 @@ export class MockDataService {
     return this.makeCopy(this.workItemLinks);
   }
 
+  public getWorkItemLinksForWorkItem(workItemId: string): any {
+    var result: any = [];
+    for (var i = 0; i < this.workItemLinks.length; i++)
+      if (this.workItemLinks[i].relationships.source.data.id === workItemId) {
+        result.push(this.makeCopy(this.workItemLinks[i]));
```

Comment:
I saw how did you implement makeCopy, but that'll fail in some cases. I'd suggest to use cloneDeep from lodash.